clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -3604,6 +3604,45 @@ def IsConstantOp : CIR_Op<"is_constant", [Pure]> {
   }];
 }
 
+
+def CIR_SwitchFlatOp : CIR_Op<"switch.flat", [AttrSizedOperandSegments, Terminator]> {
```

Comment:
`CIR_SwitchFlatOp` -> `SwitchFlatOp`, not sure why you are prefixing it with `CIR_`, we don't do that unless it's a tablegen class thing.